src/utils/issue.ts

```diff
@@ -112,11 +104,42 @@ export async function closePullRequestForAnIssue(context: Context, issueNumber:
   return logger.info(comment);
 }
 
+async function confirmMultiAssignment(context: Context, issueNumber: number, usernames: string[]) {
+  const { logger, payload, octokit } = context;
+
+  if (usernames.length < 2) {
+    return;
+  }
+
+  const { private: isPrivate } = payload.repository;
+
+  const {
+    data: { assignees },
+  } = await octokit.issues.get({
+    owner: payload.repository.owner.login,
+    repo: payload.repository.name,
+    issue_number: issueNumber,
+  });
+
+  if (!assignees?.length) {
+    const log = logger.error("We detected that this task was not assigned to anyone. Please report this to the maintainers.", { issueNumber, usernames });
+    await addCommentToIssue(context, log?.logMessage.diff as string);
+    throw new Error(log?.logMessage.raw);
+  }
+
+  if (isPrivate && assignees?.length <= 1) {
+    const log = logger.error("This task belongs to a private repo and can only be assigned to one user without an official paid GitHub subscription.", {
+      issueNumber,
+    });
+    await addCommentToIssue(context, log?.logMessage.diff as string);
+  }
+}
+
 export async function addAssignees(context: Context, issueNo: number, assignees: string[]) {
   const payload = context.payload;
 
   try {
-    await context.octokit.rest.issues.addAssignees({
+    await context.octokit.issues.addAssignees({
```

Comment:
Probably should use `rest`

Reply:
I thought that was purely a preference thing since `octokit` wraps all of the modules but I see the pros of using `rest`, will do this from now on

Reply:
Had a talk with @whilefoo and `rest` seems to be the modern way to use it, the other one being the legacy.